### PR TITLE
Allow templates from all namespaces to be processed

### PR DESCRIPTION
**Pull request by RoniKishner**
##### Short description:
Change the API call of processedtemplates to the current template namespace
##### More details:
The previous call used "openshift" namespace only, this change allow a template from any namespace to be processed

https://github.com/kubevirt/common-templates/blob/master/templates/README.md

### CODE REVIEW

**Comment by RoniKishner** [comment]
!Verified

File: ocp_resources/template.py [hunk 1]
```diff
@@ -53,14 +53,15 @@ def process(self, client=None, **kwargs):
         instance_dict["parameters"] = params
         # TODO: remove after fix - https://issues.redhat.com/browse/KNIP-1055 (bug 1753554)
```
Comment (myakove) [inline ocp_resources/template.py:54]: Can we remove the TODO? I see that the RFE is closed.

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by RoniKishner** [comment]
!Verified

**Review by myakove — APPROVED** [review]

**Comment by myakove** [comment]
!NEED-REBASE

**Comment by myakove** [comment]
test

**Comment by myakove** [comment]
!-NEED_REBASE

**Comment by myakove** [comment]
-!NEED_REBASE

**Comment by myakove** [comment]
!-NEED-REBASE

**Review by myakove — APPROVED** [review]

**Comment by RoniKishner** [comment]
!Verified

**Comment by myakove** [comment]
@rnetser Please review

**Review by rnetser — APPROVED** [review]

**Comment by RoniKishner** [comment]
!cherry-pick v4.12

**Pull request merged**